### PR TITLE
Add coverage configuration for istanbul

### LINKED ISSUE #129: Configure coverage for web-component-tester
We can use istanbul for this, but it appears to have some incompatibilities.

### PR DESCRIPTION
**Pull request by TimvdLippe**
image

Fixes #129

### CODE REVIEW

**Comment by eanker** [comment]
Is it possible to let travis pass with istanbul enabled?

**Comment by TimvdLippe** [comment]
Yes it should ;p Taking a look now

**Comment by eanker** [comment]
How do you install it?
I get the same error ("Could not find WCT plugin named "istanbul"), when running local.

Bower and npm install didn't work

**Comment by TimvdLippe** [comment]
You  must install the dependency globally too. I have added a section to the README to clarify this.

**Comment by eanker** [comment]
Confirming it works locally now.

File: bower.json [hunk 1]
```diff
@@ -23,13 +23,13 @@
     "iron-lazy-pages": "^1.0.2",
     "Sortable": "TimvdLippe/Sortable#fix-es5-compilation",
     "marked-element": "TimvdLippe/marked-element#expose-callback",
-    "prism-element": "TimvdLippe/prism-element#fork",
+    "prism-element": "TimvdLippe/prism-element#prism-css-module",
```
Comment (eanker) [inline bower.json]: Why have you changed this in this pr?

File: bower.json [hunk 2]
```diff
@@ -23,13 +23,13 @@
...
     "polymerfire": "firebase/polymerfire#^0.9.2"
   },
   "resolutions": {
     "carbon-route": "reset-tail",
     "marked-element": "expose-callback",
-    "prism-element": "fork"
+    "prism-element": "prism-css-module"
```
Comment (txsmith) [inline bower.json:32]: What is this?

Comment (txsmith) [inline bower.json:32]: Resolved IRL

**Pull request merged**